packages/smart-contracts/scripts-create2/contract-setup/setups.ts

```diff
@@ -1,143 +1,10 @@
 import { HardhatRuntimeEnvironmentExtended } from '../types';
-import { erc20SwapConversionArtifact } from '../../src/lib';
-import { batchPaymentsArtifact } from '../../src/lib';
-import utils from '@requestnetwork/utils';
-import {
-  updateBatchPaymentFees,
-  updatePaymentErc20FeeProxy,
-  updatePaymentEthFeeProxy,
-  updateChainlinkConversionPath,
-  updateRequestSwapFees,
-  updateSwapRouter,
-} from './adminTasks';
+import { setupETHConversionProxy } from './setupETHConversionProxy';
+import { setupBatchPayments } from './setupBatchPayments';
```

Comment:
I might have missed something, to be sure, is setupBatchPayments still implemented? 
(setupERC20SwapToConversion also)

Reply:
yes